checkstyle-tester/diff.groovy

```diff
@@ -289,6 +292,38 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateConfigFileHtml(diffDir, configFile) {
+    def configFilenameNoExtension = getFilenameNoExtension(configFile.name)
+    def diffToolDir = Paths.get("").toAbsolutePath()
+        .parent
+        .resolve("patch-diff-report-tool")
+        .toFile()
+    def diffToolJarPath = getPathToDiffToolJar(diffToolDir)
+    this.class.classLoader.rootLoader.addURL(new URL("file:$diffToolJarPath"))
+
+    def textTransform = Class.forName("com.github.checkstyle.site.TextTransform").newInstance()
```

Comment:
Its probably because I don't know groovy well, but since we loaded the JAR into the environment, we can't call the class directly as a normal class? IE: `new com.github.checkstyle.site.TextTransform()`

Reply:
` 309: unable to resolve class com.github.checkstyle.site.TextTransform `

Reply:
I've tried several other approaches, with no luck.
`unable to resolve class com.github.checkstyle.site.TextTransform`

Reply:
ok, that seems slightly weird but must be some limitation of groovy.